docs/rfcs/RFC0002-MonoRepo-Gardener-Rotations.md

```diff
@@ -0,0 +1,137 @@
+# Mono-repo Gardener
+
+Gardeners<sup>[1]</sup> are responsible for keeping their mono-repos shippable;
+ensuring bad commits get reverted by keeping build, test, and nightly release
+workflows green so that developers can remain productive and bugs are caught
+closer to commit times.
+
+In order to facilitate this work, gardeners have some super powers afforded to them
+that other developers do not such as ability to bypass any checks in CI.
+
+Note this concept isn't novel and can be referred to as build cops or build sheriffs
+in other projects. Good judgement and commitment are critical to being a good gardener.
```

Comment:
nit: could wrap the other terms in quotes to highlight them (or italicize) 
```suggestion
Note this concept isn't novel and can be referred to as "build cops" or "build sheriffs"
in other projects. Good judgement and commitment are critical to being a good gardener.
```

(ah, and this was forked from a playbook that I wrote, funny reviewing docs I partly wrote myself 😅)